### PR TITLE
Update project dependencies

### PR DESCRIPTION
**Pull request by cathyzoller**
@djkirby 
The latest version of redbox-react might be useable if added to webpack plugins directly, rather than from within .babelrc, but I have my doubts, so just removed it altogether.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,9 +28,8 @@
     "eslint-plugin-react": "^2.3.0",
     "express": "^4.13.3",
     "node-sass": "^3.4.2",
-    "react-transform-catch-errors": "^1.0.0",
-    "react-transform-hmr": "^1.0.0",
-    "redbox-react": "^1.0.1",
+    "react-transform-catch-errors": "^1.0.2",
```
Comment (djkirby) [inline package.json]: do we still need this dependency?

File: package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "pivotal-swimlanes",
   "description": "Swimlane view for current Pivotal sprint",
   "engines": {
-    "node": "4.0.0",
+    "node": "6.5.0",
```
Comment (djkirby) [inline package.json]: I agree with this, could you put it in a separate commit though? Just for clarity if going through the commit log

File: package.json [hunk 3]
```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "clean": "rimraf dist",
     "build:webpack": "NODE_ENV=production $(npm bin)/webpack --config webpack.config.prod.js",
-    "build": "npm run clean && npm run build:webpack",
+    "build": "npm run clean && npm cache clear && npm rebuild node-sass && npm run build:webpack",
```
Comment (djkirby) [inline package.json]: Just one thing, do you think we could move the `npm cache clear && npm rebuild node-sass` to the `clean` script? That stuff sounds like it's part of 'cleaning'

**Comment by cathyzoller** [comment]
@djkirby made requested changes.
